### PR TITLE
Subaru: longitudinal incorrect "inactive" rpm

### LINKED ISSUE #29487: Subaru longitudinal: transmission fault on Crosstrek 2018
### Describe the bug

I tested the recently merged Subaru gen1 experimental long (https://github.com/commaai/openpilot/pull/28872) on Crosstrek 2018 and I am getting transmission fault on ignition, after safety mode is set. Route: `8de015561e1ea4a0|2023-08-19--16-50-12`

I managed to narrow down the issue by checking out https://github.com/commaai/openpilot/pull/28872/commits/80d634cf474521f0bd54ba34256e9382ab60f259 in locally cached feature-subaru-long branch. This fixed the transmission fault on ignition, openpilot engaged but eyesight faulted when braking. Route: `8de015561e1ea4a0|2023-08-19--21-58-10`

After reverting https://github.com/commaai/openpilot/pull/28872/commits/44f493fdf6a20ebedea320ee09044c4071a04810 eyesight did not fault when braking and I managed to finish one of my regular test routes without issues (~5min with some slowing down behind lead cars). Route: `8de015561e1ea4a0|2023-08-19--22-40-39`

### Which car does this affect?

Subaru Crosstrek 2018

### Provide a route where the issue occurs

8de015561e1ea4a0|2023-08-19--16-50-12

### openpilot version

012060ba328da69e7fd62531bd65e8d064b3f6c0

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by jnewb1**
the "inactive" rpm should be zero, not 600 which is just a good base RPM for zero acceleration. caused transmission fault here: https://github.com/commaai/openpilot/issues/29487

Resolves: https://github.com/commaai/openpilot/issues/29487

### CODE REVIEW

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -41,13 +41,14 @@ def __init__(self, CP):
   RPM_MIN = 0
   RPM_MAX = 2400
 
-  RPM_INACTIVE = 600             # a good base rpm for zero acceleration
+  RPM_INACTIVE = 0
+  RPM_BASE = 600             # a good base rpm for zero acceleration
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:45]: Copy what gm calls this

**Comment by martinl** [comment]
I can confirm transmission fault is resolved. Eyesight still faulted after a few minutes engaged but I think this can be resolved with a different PR. Route: `8de015561e1ea4a0|2023-08-21--12-09-49`.

File: selfdrive/car/subaru/carcontroller.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def update(self, CC, CS, now_nanos):
       cruise_brake = clip(apply_brake, CarControllerParams.BRAKE_MIN, CarControllerParams.BRAKE_MAX)
     else:
       cruise_throttle = CarControllerParams.THROTTLE_INACTIVE
-      cruise_rpm = CarControllerParams.RPM_INACTIVE
+      cruise_rpm = CarControllerParams.RPM_MIN
```
Comment (sshane) [inline selfdrive/car/subaru/carcontroller.py:59]: I wouldn't call the other value `RPM_INACTIVE`, as that matches the safety value. GM calls it `ZERO_GAS`

Comment (jnewb1) [reply selfdrive/car/subaru/carcontroller.py:59]: I'll merge the fix and then standardize the names in another PR

**Pull request merged**